app/views/ems_container/_show_dashboard.html.haml

```diff
@@ -4,8 +4,9 @@
     .row.row-tile-pf
       = react('RefreshDataNotification', :providerId => @record.id.to_s, 
        :apiUrl => 'container_dashboard/refresh_status_data')
-  .row.row-tile-pf
-    = react('ContainerDashboardCards', :providerId => @record.id.to_s)
+  - if @record
+    .row.row-tile-pf
+      = react('ContainerDashboardCards', :providerId => @record.id.to_s)
```

Comment:
Is this an unsaved record?  How can it have no id?

Reply:
it's not that is has no id - it's that `@record` is nil - real question is how can it be nil on the _show_ page.

Reply:
oh wait, @record isn't set... I misread it.

It's a shame we have so many `if @record` checks in this file.

Reply:
I believe this is a dashboard across all container providers?  Or is this the dashboard for a single provider?

Reply:
yeah, @DavidResende0 do you have any idea how this was rendered with a nil object?

Reply:
Not sure, it seems that `ContainerDashboardCards` displays a summary of all providers regardless of whether an id is submitted or not
![image](https://user-images.githubusercontent.com/64800041/189406872-72bc981a-0043-480a-8550-1c50fb125584.png)

Reply:
here nodes for example, is showing all nodes across all container providers in manageiq

Reply:
looking at https://github.com/ManageIQ/manageiq-ui-classic/pull/8241/files#diff-78683e45ac4fd8172edf40ede343764eb6183579defb7a518073173e1a414fb3, it seems submitting without an id was the solution in the past @Fryguy @jrafanie